src/grt/src/fastroute/CMakeLists.txt

```diff
@@ -58,5 +58,6 @@ target_link_libraries(FastRoute4.1
     utl_lib
     stt_lib
     odb
+    dbSta_lib
```

Comment:
Why is this needed?

Reply:
It is needed to include the definition of slack, but if needed I can change it to be just float instead.